pkg/kubelet/network/cni/cni.go

```diff
@@ -152,6 +152,11 @@ func (plugin *cniNetworkPlugin) Status(namespace string, name string, id kubecon
 	return &network.PodNetworkStatus{IP: ip}, nil
 }
 
+// Return the network name of the CNI plugin.
```

Comment:
nit: godoc doens't start with the name, and `Networkname` if we care about avoiding `get` names